### ISSUE TITLE
Accessibility improvements on Select and option component

### ISSUE THREAD

**Issue by MehmetCanBOZ** [Trendyol/baklava#587]
Select and Option component does not have role and aria-selected attribute.
We can add "listbox" and "option" roles to this component.

**Comment by muratcorlu** [comment]
Hi @MehmetCanBOZ. Nice catch! Thanks for reporting. Let's have a look at it.

Please mention if you want to work on this, with a PR.

**Comment by muratcorlu** [comment]
Oh, I just noticed you already created a PR. 😄  🚀
